invenio_app/limiter.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Flask Limiter functions."""
+
+from __future__ import absolute_import, print_function
+
+import pkg_resources
+from flask import current_app, request
+
+
+def limit_with_useragent_and_ip():
```

Comment:
I think we should move limit to the end of the function name, otherwise, my first reaction when I read it it's that this does some kind of limiting, but it doesn't. It just creates a key which eventually will be used to keep track of the limit for a certain entity (in our case a user). What do you think about something like `useragent_and_ip_limit_key`?

Reply:
Sounds good to me.